testing/test_junitxml.py

```diff
@@ -1046,7 +1046,9 @@ def test_x(i):
             assert i != 22
     """
     )
-    _, dom = runandparse(testdir, "-n2")
+    # XXX: why does "-p xdist" work here, but xdist.plugin is required with
+    # other tests (to recognize the pytest_configure hook in there)?!
+    _, dom = runandparse(testdir, "-p xdist -n2")
```

Comment:
Any idea about this?
I've assumed that `-p xdist` would work in general, but it looks like if it was loaded in the outer pytest already `xdist` does not contain the `pytest_configure` hook - likely because the entrypoint mapping is not considered anymore (from "xdist" to "xdist.plugin") - but why does it work here then?

Reply:
Yeah.. but why is "-p xdist" enough here?

Reply:
https://github.com/pytest-dev/pytest-xdist/blob/master/setup.py#L19 - it appears the xdist core plugin is named by entrypoint

Reply:
What confuses me here is that `-p xdist.plugin` is required in other places here, but `-p xdist` here.  Likely some internal sys.path thing or similar..